roles/servicetelemetry/tasks/_local_signing_authority.yml

```diff
@@ -2,7 +2,7 @@
 - name: Create selfsigned Issuer
   k8s:
     definition:
-      apiVersion: certmanager.k8s.io/v1alpha1
+      apiVersion: cert-manager.io/v1
```

Comment:
Also updating for certificate creation for the Elasticsearch deployment. Honestly this should probably just be moved to the `component_elasticsearch.yml` and put into a `block` much like I did for the `component_qdr.yml`. It's unnecessary though, and I would only do this as a follow up later since it's not required to provide minimal changes.